### PR TITLE
docs(awss3): long term retention tutorial

### LINKED ISSUE #5033: How to export data to s3 or other cloud storage from otel collector
Users want to export data to s3 for archiving purposes

### PR DESCRIPTION
**Pull request by prashant-shahi**
Fixes https://github.com/SigNoz/signoz/issues/5033

This guide has been asked by number of users running SigNoz who want to retain data for longer duration due to the compliance reasons. And not necessary have a requirement to query the retained data in SigNoz.

Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

File: docs/tutorial/infinite-retention-aws-s3.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+It is a common practice to backup any data for longer durations due to
+compliance and audit purposes. You can use [AWS S3 Exporter][1] to
+retain the OpenTelemetry data as long as we need.
+
+## Limitation of AWS S3 Exporter
+
+- Data in AWS S3 is inaccessible in SigNoz UI
+- Need to use third-party tool like **Amazon Athena** to query data
+- Manual task for adding prefixes for different types of signals like logs,
```
Comment (makeavish) [inline docs/tutorial/infinite-retention-aws-s3.md]: I do not understand this point, can you please explain?

File: docs/tutorial/infinite-retention-aws-s3.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+- In case of **SigNoz Self-hosted**, you can add it in SigNoz OtelCollector
+configuration to retain all incoming data of `logs` pipelines.
+- In case of **SigNoz Cloud**, you will need to update all OtelCollector
+configuration that is sending data.
+- For **K8s-Infra** helm chart, you can add it in the `otelAgent` configuration.
+- For any **Standalone OtelCollector** deployments that is directly sending
+data to SigNoz, you can add it in its respective configuration.
+
+List of all supported configuration of `awss3` exporter can be found [here][4].
```
Comment (makeavish) [inline docs/tutorial/infinite-retention-aws-s3.md]: Not sure if link is correct

File: docs/tutorial/infinite-retention-aws-s3.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Limitation of AWS S3 Exporter
+
+- Data in AWS S3 is inaccessible in SigNoz UI
+- Need to use third-party tool like **Amazon Athena** to query data
+- Manual task for adding prefixes for different types of signals like logs,
+  metrics, traces
+
+:::info
+If you want to query data stored in AWS S3 using **SigNoz** and do not
```
Comment (makeavish) [inline docs/tutorial/infinite-retention-aws-s3.md:22]: Link is incorrect

File: docs/tutorial/infinite-retention-aws-s3.md (hunk 1 above)
Comment (prashant-shahi) [reply docs/tutorial/infinite-retention-aws-s3.md]: Users won't be able to query the data in S3 using SigNoz UI (if they have any expectation as such, that would be cleared here).
And if there comes a time when they need to access the data, they could do it using tools like **Amazon Athena** or **Google BigQuery**.

> - Manual task for adding prefixes for different types of signals like logs, ...

This last point is mostly to point at the current limitation of the `awss3` exporter where users have to prefix manually to separate data from logs/metrics/traces.

Comment (makeavish) [reply docs/tutorial/infinite-retention-aws-s3.md]: Can you just this explanation for last point to doc as well?
Like - Due to limitation on S3exporter you won't be able differentiate different signals

File: docs/tutorial/infinite-retention-aws-s3.md (hunk 3 above)
Comment (prashant-shahi) [reply docs/tutorial/infinite-retention-aws-s3.md:22]: updated

File: docs/tutorial/infinite-retention-aws-s3.md (hunk 2 above)
Comment (prashant-shahi) [reply docs/tutorial/infinite-retention-aws-s3.md]: updated

**Review by makeavish — DISMISSED** [review]

**Review by Calm-Rock — DISMISSED** [review]

File: docs/tutorial/infinite-retention-aws-s3.md (hunk 1 above)
Comment (Calm-Rock) [reply docs/tutorial/infinite-retention-aws-s3.md]: @prashant-shahi please resolve this conversation.
Will merge this PR and migrate it to the new website post that.

Comment (prashant-shahi) [reply docs/tutorial/infinite-retention-aws-s3.md]: Updated

**Review by Calm-Rock — APPROVED** [review]

**Pull request merged**